.gitlab-ci.yml

```diff
@@ -27,8 +27,9 @@ check_dependencies:
   tags: 
     - docker
   script:
-    - git clone https://github.com/pcengines/pce-fw-builder.git -b rel_v1.0.0 /home/coreboot/pce-fw-builder
+    - git clone https://github.com/pcengines/pce-fw-builder.git /home/coreboot/pce-fw-builder
```

Comment:
@miczyg1 this means we are on master, I would like to be on s

Reply:
@pietrushnic look below, I'm checking out on the latest tag